roads.mss

```diff
@@ -3100,3 +3100,81 @@ tertiary is rendered from z10 and is not included in osm_planet_roads. */
     }
   }
 }
+#railways-text-name {
+  /* Others started from z15. */
```

Comment:
I think subway, tram and other city transport labels should be rendered much later (like z18-z19), since such cities tend to be cluttered with labels anyway.